drivers/adf4350/adf4350.c

```diff
@@ -248,7 +248,7 @@ int64_t adf4350_set_freq(adf4350_dev *dev,
 		(dev->pdata->r2_user_settings & (ADF4350_REG2_PD_POLARITY_POS |
 		ADF4350_REG2_LDP_6ns | ADF4350_REG2_LDF_INT_N |
 		ADF4350_REG2_CHARGE_PUMP_CURR_uA(5000) |
-		ADF4350_REG2_MUXOUT(0x7) | ADF4350_REG2_NOISE_MODE(0x9)));
+		ADF4350_REG2_MUXOUT(0x7) | ADF4350_REG2_NOISE_MODE(0b101)));
```

Comment:
ADF4350_REG2_NOISE_MODE has a length of 2 bits, so 0b101 value is incorrect. I would also specify this in its definition (e.g., (((x) & 0x3) << 29)).
In this place, ADF4350_REG2_NOISE_MODE should be set 0x3 to enable the low spur mode.